changelog/1.17.0.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "1.17.0"
+description: "Released on 03/17/2021"
+---
+
+### Breaking Changes ❗
+
+- (?) Wsp orgs migration
+
+### Features ✨
+
+- Environment Providers **TODO**
+- Autostart: Autostart allows you to set the time when Coder
+  automatically starts and builds your environments.
+  - web: A new onboarding wizard is available for setting a preferred Autostart
+    time.
+  - web: Visit your account preferences > Autostart to set a time and any
+  environments that should be automatically started. We recommend enabling this
+  feature for the environments you use the most.
+  - web: Autostart action has been added to the Audit Log
+  - web: A new checkbox for opting into Autostart is available on the Create and
+    Edit Environment modals.
+
+- web: Environment stop actions have been added to the Audit Log
+- web: Show most recent Build Log for offline environments
+- cli: Improved sign in flow with (cdr/coder-cli#274)
+- web: Improved error messages within the Build Log
+  - Improve messages when '/coder/configure' is not executable
+- code-web: Upgrade to code-server version 3.9.0
+- jetbrains: The Mac OS keymap plugin is now automatically installed
+- web: Resources section on Environments page has been redesigned
+- web: Apply consistent design theme across all Preferences pages
+
+### Bug Fixes 🐛
+
+- web: Fix DevURL error page links using incorrect URIs.
+- infra: Fix timeout when deleting users with many environments
+- infra: Increase offline timeout in trial versions of Coder
+- web: Use proper casing when referencing 'GitHub' and 'GitLab'
+- infra: Fix renaming an OAuth provider without creating a duplicate. Migrating
+  to 1.17.0 will remove these duplicates. They are not visible in the UI.
```

Comment:
cc @Emyrk to verify

Reply:
The language feels odd here, but it is correct.
> Fix renaming an OAuth provider without creating a duplicate.

Maybe??

Fix renaming an OAuth provider to correctly rename the existing OAuth provider without creating a duplicate.